### ISSUE TITLE
asciinema files are missing from public instance ptpb.pw

### ISSUE THREAD

**Issue by nheir** [ptpb/pb#173]
Hello,
static/hack/asciinema.{css,js} are missing from https://ptpb.pw so the terminal handler does not work.

**Comment by buhman** [comment]
Hmm.

```
zack@balin ~
% aws s3 ls ptpb/static/hack/
2016-10-14 11:16:49          0 
2016-10-14 11:17:05      52367 asciinema.css
2016-10-14 11:17:06     455761 asciinema.js
2016-10-14 11:17:06        148 hack.css
```

```
zack@balin ~
% curl -s https://d34zelngniy2d8.cloudfront.net/static/hack/asciinema.css | head 
.asciinema-player-wrapper {
  position: relative;
  text-align: center;
  outline: none;
}
.asciinema-player-wrapper .title-bar {
  display: none;
  top: -78px;
  transition: top 0.15s linear;
  position: absolute;
```

Looks good to me; I'll check this again when @silverp1 fixes #174.

**Comment by buhman** [comment]
Ah, nevermind, the template is incorrect. Fixed in 29f64b08067a9e31865055e336fbe32fd1e61ebf.

**Comment by buhman** [comment]
Deployed. If you look at the source of http://pb2-887724492.us-east-1.elb.amazonaws.com/t/hLdj, you'll see the correct CDN url in use.
